en/02_Developer_Guides/08_Performance/03_HTTP_Cache_Headers.md

```diff
@@ -11,10 +11,11 @@ that the response should be not considered cacheable.
 HTTP caches can either be intermediary caches (e.g. CDNs and proxies), or clients (e.g. browsers).
```

Comment:
This page has been renamed because I think it makes sense to display this after the template-caching/object-caching/cache-adapters stuff which is all related and is "cache this portion of data", and display it closer to static publishing which is related because that and http cache headers are "Cache the whole page"